resources/scss/ceres/widgets/Helper/_base-widget.scss

```diff
@@ -51,26 +51,48 @@
     }
 }
 
-#page-header-parent > .widget {
-    max-width: 1200px;
-    width: 100%;
-    background-color: $gray-lightest;
-    
-    &:not(.widget-grid) {
-        max-width: 1170px;
+#page-header-parent {
+
+    > .widget {
+        max-width: 1200px;
+        min-width: 100%;
+        background-color: $gray-lightest;
+        left: 15px;
+        right: 15px;
+
+        &:not(.widget-grid) {
+            max-width: 1170px;
+            min-width: inherit;
+        }
     }
 }
 
-header .widget
-{
-    margin-bottom: 0;
+
+html.scrollbar .widget-fw {
```

Comment:
Die Klasse`.scrollbar` sorgt dafür, dass das raster nicht mehr korrekt ausgerichtet ist:

**html.scrollbar**
![Bildschirmfoto 2019-10-23 um 08 27 11](https://user-images.githubusercontent.com/7404578/67364021-01177400-f56f-11e9-9d39-e5e5254dbac1.png)

**html:not(.scrollbar)**
![Bildschirmfoto 2019-10-23 um 08 27 04](https://user-images.githubusercontent.com/7404578/67364061-11c7ea00-f56f-11e9-8299-0d45e81a566a.png)

Reply:
Lässt sich nicht nachstellen